sources/packages/api/src-sql/InstitutionUsers/Add-col-institution-user-guid.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE "institution_users" ADD COLUMN IF NOT EXISTS "guid" VARCHAR(100) NULL;
+
+COMMENT ON COLUMN institution_users.guid IS 'The guid colum to store BCeID GUID. This value will be used to create advance user in system';
```

Comment:
I liked the comment 👍 
Please just add something to indicate that it is the user guid.